src/components/OperationsList/Operation.js

```diff
@@ -49,6 +49,10 @@ const OperationRaw = styled(Box).attrs({
 `
 
 const Address = ({ value }: { value: string }) => {
+    if (!value) {
+      return <Box />
```

Comment:
I usually rather neuter a component into a stub, than removing it completely from DOM to prevent side effects. But, yeah, in this context `null` is perfectly safe and would have made more sense.